maxim/logger/langchain/utils.py

```
@@ -699,7 +790,8 @@ def parse_langchain_messages(
                                     "content": "",
                                 }
                             )
-        return messages
+        return messages, attachments
     except Exception as e:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)